gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -230,11 +230,18 @@ MM_VerboseHandlerOutput::handleInitializedRegion(J9HookInterface** hook, uintptr
 	MM_InitializedEvent* event = (MM_InitializedEvent*)eventData;
 	MM_VerboseWriterChain* writer = _manager->getWriterChain();
 	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(event->currentThread);
+#if defined(OMR_GC_VLHGC) && defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)
+	const char *arrayletDoubleMappingStatus = event->isArrayletDoubleMappingEnabled ? "enabled" : "disabled";
+	const char *arrayletDoubleMappingRequested = _extensions->isArrayletDoubleMapRequested ? "true" : "false";
+#endif /* OMR_GC_VLHGC && OMR_GC_DOUBLE_MAP_ARRAYLETS */
 
 	writer->formatAndOutput(env, 1, "<region>");
 	writer->formatAndOutput(env, 2, "<attribute name=\"regionSize\" value=\"%zu\" />", event->regionSize);
 	writer->formatAndOutput(env, 2, "<attribute name=\"regionCount\" value=\"%zu\" />", event->regionCount);
 	writer->formatAndOutput(env, 2, "<attribute name=\"arrayletLeafSize\" value=\"%zu\" />", event->arrayletLeafSize);
+#if defined(OMR_GC_VLHGC) && defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)
+	writer->formatAndOutput(env, 2, "<attribute name=\"arrayletDoubleMapping\" requested=\"%s\" status=\"%s\"/>", arrayletDoubleMappingRequested, arrayletDoubleMappingStatus);
```

Comment:
runtime isVLHGC() check is also needed

Reply:
you need to modify schema.xml

Reply:
if arrayletDoubleMappingRequested is 'false', it makes little sense to report arrayletDoubleMappingStatus (it better 'disabled'). you'll need to formatAndOutput lines distiguished by the value of arrayletDoubleMappingRequested.

in schema.xml you specify that arrayletDoubleMappingStatus  is an optional attribute.